src-electron/generator/matter/app/zap-templates/templates/chip/helper.js

```diff
@@ -745,6 +745,97 @@ async function chip_access_elements(options) {
   return templateUtil.templatePromise(this.global, p);
 }
 
+/**
+ * If helper that checks if an attribute is basic or not based on language
+ * being generated for.
+ * For eg: In java, an attribute is not basic if it is either nullable, optional,
+ * array type or struct.
+ * Note: This helper should be used within an attribute block helper.
+ * Available Options:
+ * - language: Determine basic attribute based on language.
+ * example:
+ * {{#if_basic_attribute type}}
+ * type is basic
+ * {{else}}
+ * type is not basic
+ * {{/if_basic_attribute}}
+ * @param {*} type
+ * @param {*} clusterId
+ * @param {*} options
+ * @returns Promise of content
+ */
+async function if_basic_attribute(type, clusterId, options) {
```

Comment:
This is Matter-java-codegen-specific, right?  Including some Matter-java-codegen specific definition of "basic"?

Please put this in the java helper file, not here.

Reply:
This is where the corresponding deprecated helper is(if_basic_global_response). Do you still want this in a newly created java helper file?

Reply:
I think so, yes.  The if_basic_global_response thing was mis-placed to start with...

Reply:
Moved to java helper.js